firmware/V1.x_V2.x/SAMD21E17/LoRa-CLI/LoRa-CLI.ino

```diff
@@ -95,7 +97,7 @@ void setup() {
   Serial.print(F("[SX1262] Initializing ... "));
   //debing(float freq, float bw, uint8_t sf, uint8_t cr, uint8_t syncWord, int8_t power, uint16_t preambleLength, float tcxoVoltage, bool useRegulatorLDO)
 
-  int state = radio.begin(915.0, 250, 7, 5, 0x12, 20, 8, 0, false);
+  int state = radio.begin(915.0, 125.0, 7, 5, 0x12, 20, 8, 0, false);
```

Comment:
```suggestion
  float freq = 915.0;
  float bandwidth = 125.0;
  uint8_t spreadingFactor = 7;
  uint8_t codingRate = 5;
  uint8_t syncWord = RADIOLIB_SX126X_SYNC_WORD_PRIVATE;
  int8_t power = 20;
  uint16_t preambleLength = 8;
  float tcxoVoltage = 0;
  bool useRegulatorLDO = false;
  int state = radio.begin(freq, bandwidth, spreadingFactor, codingRate, syncWord, power, preambleLength, tcxoVoltage, useRegulatorLDO);
```

Reply:
I would avoid using magic numbers, instead I suggest to create variables for each param